policies/design-process.md

```diff
@@ -0,0 +1,141 @@
+Policy for Creating Designs
+===========================
+
+From time to time the OTC may require that certain problems should create a
+design document before implementation should start. In that case this policy
+will apply.
+
+The design approach described in this document should be considered iterative.
+Only "just enough" design should be done to get the implementation project to
+the next stage. Certain design decisions and sub-problems may be deferred until
+some later stage.
+
+Inputs
+------
+
+Prior to commencing design work the OTC shall identify:
+
+1) The business requirements for the problem to be solved (obtained from the OMC)
+2) The scope of the specific problem area to be addressed
+3) Any additional design constraints that the OTC wishes to impose
+4) Any other constraints that may apply (e.g. timescales)
+5) The individual(s) that will work on developing the design (refered to as the
+   "design team" in this document)
+
+Typically these will be identified during an OTC meeting and documented in the
+meeting minutes.
+
+Outputs
+-------
+
+The result of this process should be a design document in markdown format,
+reviewed via a pull request and approved for merge into the otc git repository.
+
+Other outputs may include feedback on the requirements and constraints that
+were the input into the process. For example some candidate solutions that are
```

Comment:
comma after _some_?

Reply:
I assume you really meant comma after "For example". Added.